packages/shared/src/db/types.ts

```diff
@@ -100,6 +96,21 @@ export type Settings = BaseModel<'settings'>;
 export type PostWindow = BaseModel<'postWindows'>;
 export type VolumeSettings = BaseModel<'volumeSettings'>;
 
+export type Chat = {
```

Comment:
Yeah, essentially a chat is "a thing that goes in the chat list". Agree this could be cleaned up, but I think this refactor is an incremental step toward a better soluton.